crates/autopilot/src/database.rs

```diff
@@ -24,13 +24,23 @@ impl Postgres {
         Ok(Self(PgPool::connect(url).await?))
     }
 
-    pub async fn update_table_rows_metric(&self) -> sqlx::Result<()> {
+    pub async fn update_database_metrics(&self) -> sqlx::Result<()> {
         let metrics = Metrics::get();
+
+        // update table row metrics
         for &table in database::ALL_TABLES {
             let mut ex = self.0.acquire().await?;
             let count = count_rows_in_table(&mut ex, table).await?;
             metrics.table_rows.with_label_values(&[table]).set(count);
         }
+
+        // update unused app data metric
+        {
+            let mut ex = self.0.acquire().await?;
```

Comment:
This pattern already existed before but wouldn't it make sense to only acquire a single connection and reuse it for all updates here?

Reply:
Since this doesn't start a transaction I think there should be no locks involved.
On the other hand it doesn't matter that much. Just looked weird to me.

Reply:
I think that was it.